simulate-event.js

```diff
@@ -340,7 +339,7 @@ module.exports = function (element, type, options) {
   options = extend({
     bubbles: true,
     cancelable: true
-  }, eventOptions[eventType](element, type, options), options)
+  }, eventOptions[eventType](type, options), options)
```

Comment:
Not sure, this is more from an old PR - didn't you add the `new Constructor`? Feel free to merge when you want.